GOVERNANCE.md

```diff
@@ -0,0 +1,131 @@
+# Fn Project Governance
+
+This document describes the rules and governance of the project. It is meant to be followed by all the developers of the project and the Fn Project community. Common terminology used in this governance document are listed below:
+  * Team members: members of the [Fn Team group on forums.fnproject.io](https://forums.fnproject.io/groups/FnTeam).
+  * Maintainers: Maintainers lead an individual project or parts thereof.
+  * Projects: A single repository in the [Fn Project GitHub organization](https://github.com/fnproject).
+  * The Fn Project: The sum of all activities performed under this governance, concerning one or more repositories or the community. 
+
+## Values
+The Fn Project exists to serve the CNCF ecosystem. Thus developers and community are expected to follow the values defined in the [CNCF Charter](https://www.cncf.io/about/charter/), including the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md). Furthermore, the Fn Project community strives for kindness, giving feedback effectively, and building a welcoming environment. The Fn Project developers generally decide by consensus and only resort to conflict resolution by a majority vote if consensus cannot be reached.
```

Comment:
> The Fn Project exists to serve the CNCF ecosystem.

Is this true? That seems a rather narrower scope than I thought. e.g. the project exists in a big way to be the basis for a hosted service on OCI.  I absolutely agree with the comments about the charter and CoC and the rest of the paragraph.

Reply:
'among other ecosystems' at least and not just solely, I agree it's a bit pigeon-holey even if we were at present a CNCF member project.

additionally: 

* nobody is going to read the CNCF charter, it's too long and doesn't seem like a valuable inclusion at this point in time, including but not limited to imposing rules that we aren't aware we're imposing and making it hard to discern what our actual process is in the fn project (the scope of CNCF is much larger) -- even if read literally here, readers are only expected to extract the 'values' outlined within, however the scope of that document is much larger and if any of those values are... valuable (forgive me)... perhaps we should simply inline them here. 
* the CNCF code of conduct looks mostly reasonable and handles a class of issues very effectively, though I don't like how conveniently left open to interpretation some of its finer points are they can be used rather 'conveniently' against people (e.g. historically speaking, defining `Other unethical or unprofessional conduct.` is not so easy and is a too-convenient 'catch all' imo). Since we are not a member at present, I doubt the CNCF would be an arbiter on our behalf for CoC disputes, and we should likely make our own document derivative of http://contributor-covenant.org/version/1/2/0/ with our own point of contact to handle dispute resolution as opposed to linking to the CNCF one. I agree with the overall premise of having a code of conduct as it strives for higher inclusivity, thanks for adding it.

Reply:
I agree with both your comments @rdallman. For now I've created a CODE_OF_CONDUCT.md from the contributors covenant and removed reference to CNCF.